src/episervertemplating/modules/_protected/InstantTemplates/ClientResources/Scripts/ContentTypeList.js

```diff
@@ -54,18 +54,38 @@ function (
         // contentLink: [public] String
         //     The contentlink for which the list will be filtered on.
         contentLink: null,
-
         templatesRoot: null,
+        constructor: function(){
+            var vm = this;
+            topic.subscribe("/epi/shell/action/changeview", function (name, args) {
```

Comment:
I don't think you need to subscribe the ChangeView topic. It eventually will invoke the updateView function if it is available. So instead, we declare the updateView function, then the framework will call it with the appropriate contentLink and parentLink:

```
updateView: function (data, context, additionalParams) {
  // parentLink can be retrieved via data.parentLink
}
```